OWNERS

```diff
@@ -1,29 +1,31 @@
 reviewers:
 - ciaranRoche
 - gdbranco
-- igoihman
+- robpblake
 - oriAdler
 - pvasant
 - renan-campos
-- zgalor
+- hunterkepley
+- thomasmckay
+- den-rgb
+- marcolan018
+- willkutler
+- davidleerh
 approvers:
 - ciaranRoche
 - gdbranco
-- igoihman
 - oriAdler
 - pvasant
 - renan-campos
-- vkareh
-- zgalor
 maintainers:
 - ciaranRoche
 - gdbranco
-- igoihman
 - oriAdler
 - pvasant
 - renan-campos
-- vkareh
-- zgalor
 emeritus_approvers:
 - jharrington22
 - tbrisker
```

Comment:
can remove tomer, as he is not in red hat anymore

Reply:
@ciaranRoche Dropped Tomer. Added Victor back to approvers/reviewers.